internal/engine/engine.go

```diff
@@ -797,8 +800,24 @@ func InitializeContainerChecks(ctx context.Context, p policy.Policy, cfg Contain
 			containerpol.NewHasUniqueTagCheck(cfg.DockerConfig),
 			&containerpol.MaxLayersCheck{},
 			&containerpol.HasRequiredLabelsCheck{},
+			&containerpol.HasProhibitedLabelsCheck{},
 			&containerpol.HasProhibitedContainerName{},
 		}, nil
+	case policy.PolicyKonflux:
+		return []check.Check{
+			&containerpol.HasLicenseCheck{},
+			containerpol.NewHasUniqueTagCheck(cfg.DockerConfig),
+			&containerpol.MaxLayersCheck{},
+			&containerpol.HasNoProhibitedPackagesCheck{},
+			&containerpol.HasRequiredLabelsCheck{},
+			&containerpol.RunAsNonRootCheck{},
+			&containerpol.HasModifiedFilesCheck{},
+			containerpol.NewBasedOnUbiCheck(pyxis.NewPyxisClient(
```

Comment:
Is this one of those checks that Konflux may want to opt out of?

Reply:
I think that's out of scope for this, if an image is UBI then none of these checks apply so the caller in konflux can remove preflight from their pipeline.